doc/archetypes/section-bundle/_index.md

```diff
@@ -92,6 +92,19 @@ Thou shalt always use the shortcode {{% tts %}} when referring to this product.
 
 For documentation that requires the Command Line Interface, use the {{% cli-only %}} shortcode.
 
+## User Variables
+
+For variables which a user must replace, use a command to export the variable in the shell, if possible.
+
+```bash
+$ export GTW_ID="your-gateway-id"
+$ export LNS_KEY="your-lns-api-key"
+$ export SECRET=$(echo $LNS_KEY | xxd -p -u | perl -pe 's/\n//')
+$ ttn-lw-cli gateways update $GTW_ID --lbs-lns-secret.value $SECRET
```

Comment:
Technically you don't have to export them, you can omit `export` here and declare the vars directly. But I'm fine with this too.

Reply:
Ah I wasn't aware of this https://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export

I'll get rid of the export. I don't think we need to depend on sub-processes using documentation example variables